BUILD.md

```diff
@@ -396,16 +396,18 @@ See more information on the OpenVINO Execution Provider [here](./docs/execution_
 | <code>VAD-M_FP16</code> | Intel<sup>®</sup> Vision Accelerator Design based on 8 Movidius<sup>TM</sup> MyriadX VPUs |
 | <code>VAD-F_FP32</code> | Intel<sup>®</sup> Vision Accelerator Design with an Intel<sup>®</sup> Arria<sup>®</sup> 10 FPGA |
 | <code>HETERO:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above |
+| <code>MULTI:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above |
```

Comment:
Could you add a summary description of Hetero and Multi plugins beyond supporting all silicons?

Reply:
@smkarlap In the build.md file (https://github.com/intel/onnxruntime/blob/maajid/implemet_multi_device/BUILD.md#build-instructions-6),
I have added how Hetero Device type or MULTI Device type target can be added and with some examples.

The summary of Hetero and Multi plugins is already added here (https://github.com/intel/onnxruntime/blob/maajid/implemet_multi_device/docs/execution_providers/OpenVINO-ExecutionProvider.md#heterogeneous-execution-for-openvino-ep)